vite.config.ts

```diff
@@ -1,14 +1,40 @@
 import { defineConfig } from 'vite'
+import path from 'path'
 
 // https://vitejs.dev/config/
 export default defineConfig({
   build: {
     lib: {
-      entry: 'src/my-element.ts',
+      entry: {
+        lib: path.resolve(__dirname, 'src/index.ts'),
+        app: path.resolve(__dirname, 'src/app.ts'),
+        element: path.resolve(__dirname, 'src/my-element.ts'),
+      },
+      name: 'ItkViewer',
+      fileName: (format, entryName) => {
+        const extension = format === 'es' ? '.js' : '.umd.js'
+        switch(entryName) {
+        case 'lib':
+          return `itk-viewer${extension}`
+        case 'app':
+          return `itk-viewer-app${extension}`
+        case 'element':
+          return `itk-viewer-element${extension}`
```

Comment:
Do we need to prefix?

Reply:
Yeah, I think we should just put this in another package.

Reply:
Just curious: why do we put itk-viewer in front?

Reply:
Module name corresponds to the name of the package